fpm-entrypoint.sh

```diff
@@ -3,10 +3,12 @@
 set -o errexit
 
 cd /tmp/build
+chgrp root -R /tmp/build/*
+chmod -R g=u /tmp/build/*
```

Comment:
These two probably oughtta be changed to match the `after-install.sh` ones. :)

Reply:
Are there any dot files at the top level of those directories the `/*` would miss?  This pattern should also include dotfiles at the top level containing at least two characters after the dot. :)
```suggestion
chown -R root:kong /tmp/build/{,.??}*
chmod -R g=u,g-w /tmp/build/{,.??}*
```